x-pack/plugins/apm/public/components/app/service_overview/service_overview_instances_table/intance_details.tsx

```diff
@@ -52,7 +52,7 @@ const serviceDetailsKeys = [
   SERVICE_RUNTIME_NAME,
   SERVICE_RUNTIME_VERSION,
 ];
-const containerDetailsKeys = [CONTAINER_ID, HOST_NAME, POD_NAME];
+const containerDetailsKeys = [CONTAINER_ID, HOST_HOSTNAME, POD_NAME];
```

Comment:
same here, do we need host.name || host.hostname here as well?

Reply:
I don't know, probably yes. Maybe in a different issue?